components/kyma-environment-broker/internal/process/provisioning/get_kubeconfig_step.go

```diff
@@ -49,23 +66,25 @@ func (s *GetKubeconfigStep) Run(operation internal.Operation, log logrus.FieldLo
 
 	if status.RuntimeConfiguration.Kubeconfig == nil {
 		log.Errorf("kubeconfig is not provided")
-		return operation, 1 * time.Minute, nil
+		return operation, 1 * time.Minute, nil //TODO why we try to retry after 1 minute? There is no kubeconfig!
 	}
+
 	k := *status.RuntimeConfiguration.Kubeconfig
 	hash := sha256.Sum256([]byte(k))
 	log.Infof("kubeconfig details length: %v, sha256: %v", len(k), string(hash[:]))
 	if len(k) < 10 {
-		log.Errorf("kubeconfig suspiciously small, requeueing after 30s")
+		log.Errorf("kubeconfig suspiciously small, requeueing after 30s") //TODO would it change after 30s?
```

Comment:
it was in the past that Provisioner returned not valid Kubeconfig

Reply:
You mean that afterwards it would return a proper one?

Reply:
TODOs removed